api/src/main/kotlin/edu/wgu/osmt/config/Constants.kt

```diff
@@ -1,3 +1,10 @@
 package edu.wgu.osmt.config
 
-const val UNAUTHENTICATED_USERNAME = "unauthenticated"
\ No newline at end of file
+const val UNAUTHENTICATED_USERNAME = "unauthenticated"
```

Comment:
We already apache StringUtils in the classpath. Let's reuse those instead of making our own new constants.

```
public class StringUtils {
    private static final int STRING_BUILDER_SIZE = 256;
    public static final String SPACE = " ";
    public static final String EMPTY = "";
    public static final String LF = "\n";
    public static final String CR = "\r";
    public static final int INDEX_NOT_FOUND = -1;
```